### ISSUE TITLE
A new mapfile option, no_include

### ISSUE THREAD

**Issue by LebedevRI** [include-what-you-use/include-what-you-use#472]
IWYU suggests to include a lot of `bits/` and `ext/` headers.
While that is a bug on itself, it would be super convient to be able to globally tell it not to do it by specifying something like this in the mapping file:
```
 { include: ["@<ext/.*>", "no_include"] },
```

**Comment by kimgr** [comment]
Something like that indeed looks useful. It might even be conceivable as a command-line argument, since it applies to the entire IWYU invocation -- mappings usually apply to an includer/includee pair.

**Comment by monouser7dig** [comment]
Any update on this?
Seems as if it could improve https://github.com/include-what-you-use/include-what-you-use/issues/492

**Comment by JoshuaSBrown** [comment]
I agree with the above and want to provide an example which I think demonstrates quite clearly why this would be useful. Especially when dealing with external dependencies. I am using the boost testing framework and the includes added by include-what-you-use are quite excessive. Here is the original file:

    #define BOOST_TEST_DYN_LINK                                                      
    #define BOOST_TEST_MAIN                                                          
                                                                                     
    #define BOOST_TEST_MODULE atom_test                                              
    #include <boost/test/unit_test.hpp>                                              
                                                                                 
    BOOST_AUTO_TEST_SUITE(simple_test)                                               
                                                                                 
    BOOST_AUTO_TEST_CASE(small_test) {                                                                                                                           
      int variable = 5;                                                              
      BOOST_CHECK_EQUAL(variable,5);
    }                                                                                
                                                                                 
    BOOST_AUTO_TEST_SUITE_END()        

Which compiles and runs fine. 
And this is what it looks like after I run include-what-you-use:

    #define BOOST_TEST_DYN_LINK                                                      
    #define BOOST_TEST_MAIN                                                          
                                                                                 
    #define BOOST_TEST_MODULE atom_test                                              
    #include <boost/preprocessor/arithmetic/dec.hpp>                                 
    #include <boost/preprocessor/comparison/not_equal.hpp>                           
    #include <boost/preprocessor/control/expr_iif.hpp>                               
    #include <boost/preprocessor/control/iif.hpp>                                    
    #include <boost/preprocessor/detail/auto_rec.hpp>                                
    #include <boost/preprocessor/logical/bool.hpp>                                   
    #include <boost/preprocessor/logical/compl.hpp>                                  
    #include <boost/preprocessor/repetition/detail/for.hpp>                          
    #include <boost/preprocessor/repetition/for.hpp>                                 
    #include <boost/preprocessor/seq/elem.hpp>                                       
    #include <boost/preprocessor/seq/size.hpp>                                       
    #include <boost/preprocessor/tuple/elem.hpp>                                     
    #include <boost/preprocessor/variadic/elem.hpp>                                  
    #include <boost/test/tools/old/interface.hpp>                                    
    #include <boost/test/unit_test.hpp>                                              
    #include <boost/test/unit_test_suite.hpp>                                        
    #include <boost/test/utils/basic_cstring/basic_cstring.hpp>                      
    #include <boost/test/utils/basic_cstring/basic_cstring_fwd.hpp>                  
                                                                                 
    BOOST_AUTO_TEST_SUITE(simple_test)                                               
                                                                                 
    BOOST_AUTO_TEST_CASE(small_test) {                                               
      int variable = 5;                                                              
      BOOST_CHECK_EQUAL(variable,5);                                                 
    }                                                                                
                                                                                 
    BOOST_AUTO_TEST_SUITE_END()
